src/github.com/couchbaselabs/sync_gateway/base/bucket.go

```diff
@@ -54,6 +58,10 @@ func (bucket couchbaseBucket) GetName() string {
 	return bucket.Name
 }
 
+/*func (bucket couchbaseBucket) GetUUID() string {
```

Comment:
Thanks for catching - that's a remnant from when I was trying to pass the existing bucket to cbdatasource (which expects a GetUUID() method on the bucket).